frontend/src/components/pages/club/Club.tsx

```diff
@@ -1,10 +1,62 @@
-import * as React from 'react';
+import * as React from "react";
+import { BookInfo } from "../club_display/BookInfo";
+import Box from "@material-ui/core/Box";
+import { ClubDescription } from "../club_display/ClubDescription";
+import { ContentWarnings } from "../club_display/ContentWarnings";
+import { ClubInterface } from "../../../services/backend_service_interface/backend_service_interface";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    club: {
+      width: "1500px",
+    },
+    clubContainer: {
+      alignItems: 'center',
+      display: 'flex',
+      flexDirection: 'column',
+      flexShrink: 3,
+      marginTop: theme.spacing(12),
+    },
+    contentContainer: {
+      display: 'flex',
+      flexDirection: 'column',
+      marginBottom: theme.spacing(2),
+      marginLeft: theme.spacing(3),
+      marginRight: theme.spacing(3),
+      marginTop: theme.spacing(2),
+    },
+    infoAndWarnings: {
+      display: 'flex',
+      marginBottom: theme.spacing(2),
+      marginTop: theme.spacing(1),
+    },
+  }),
+);
+
+/**
+ * Displays information of a club that the user is a member of.
+ */
+export const Club = (props) => {
+  const classes = useStyles();
+
+  const [handle, setHandle] = React.useState<string>(props.match.params.handle);
+  const [club, setClub] = React.useState<ClubInterface>(props.location.state.club);
```

Comment:
These don't need to be state. State is just for when a component needs to remember something beyond what was passed in. Since these are both just props, and neither setter function is called, the template can just refer to props.match.params.handle directly